src/main/java/it/gov/pagopa/payhub/activities/service/IuvService.java

```diff
@@ -0,0 +1,109 @@
+package it.gov.pagopa.payhub.activities.service;
+
+import it.gov.pagopa.payhub.activities.dao.IuvSequenceNumberDao;
+import it.gov.pagopa.payhub.activities.dto.OrganizationDTO;
+import it.gov.pagopa.payhub.activities.exception.ValueNotValidException;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+/**
+ * Service class exposing methods related to IUV handling.
+ */
+@Lazy
+@Service
+@Slf4j
+public class IuvService {
+
+  private static final String PROGRESSIVO_PADDING_ZEROES_SMALL = "%011d";
+  public static final String AUX_DIGIT = "3";
+
+  private final String informationSystemId;
+
+  private final IuvSequenceNumberDao progressiviVersamentoDao;
```

Comment:
lets align variable and class name (use english)

Reply:
fixed